include/mrbconf.h

```diff
@@ -204,7 +204,7 @@
 # endif
 
 # ifndef MRB_STR_BUF_MIN_SIZE
-#  define MRB_STR_BUF_MIN_SIZE 32
+#  define MRB_STR_BUF_MIN_SIZE 31
```

Comment:
This makes the default size 31 bytes. So it should be officially `31` or change here `63`. I vote for making the default value to `31`.